src/__tests__/styled.test.js

```diff
@@ -52,7 +52,7 @@ describe('styled', () => {
                 foo: 1;
             `(p)
         ).toEqual('h()');
-        expect(css).toBeCalledWith(['foo: 1']);
+        expect(css).toBeCalledWith([expect.stringContaining('foo: 1')]);
```

Comment:
this is fixing the tests, as prettier formats tagged `styled` literals to new lines, css function call won't get inline string

```diff
-styled("tag")`foo: 1`
+styled("tag")`
+  foo: 1
+`
```

3 solutions how to mitigate this:

1. using `expect.stringContaining` (this commit)
1. provide custom normalize function that removes trailing spaces + new lines
1. turn off prettier for that particular line and keep test as is

IMHO:  solution no 1, mirrors most of real life use cases of `styled` usage, thus is the best candidate for this test

Lemme know WDYT. thx

Reply:
I think it's fine as you did it. We might change it, down the road but we'll get there. 👍 Solid work!